### PR TITLE
Modify instructions to use the Kabanero stack hub

### LINKED ISSUE #40: Guide use-appsody-cli: https://github.com/acme.inc/stacks/index.yaml does not exist
Transferring issue from @nastacio from the old to new repo:

Comment:

Maybe this URL was meant as an example, but the text does not make it clear.
Making that clarification would eliminate the confusion, but still be hard to follow, so I suggest using the actual URL for Kabanero 0.6.0:

https://github.com/kabanero-io/kabanero-stack-hub/releases/download/0.6.0/kabanero-stack-hub-index.yaml

### PR DESCRIPTION
**Pull request by SueChaplain**
To address issues identified in #40 and bring the example
version levels up to the next Kabanero release.

Closes: #40

Signed-off-by: Sue Chaplain <sue_chaplain@uk.ibm.com>

### CODE REVIEW

**Comment by SueChaplain** [comment]
@jantley - these are similar changes to the stack guides, so I'm not going to trouble dev for a review. Please can you run your eye over.

**Review by jantley-ibm — APPROVED** [review]
